### PR TITLE
Let's add some (basic) Elixir support

### PR DESCRIPTION
**Pull request by vadviktor**

### CODE REVIEW

**Comment by dawikur** [comment]
Lets wait a little bit for #188 and than we can ~remove~ not-add variable (this PR was first, but it will be just a noise to first add something just to remove it in next commit). Are you fine with it?

As a side note, what terminal/font are you using? On my Arch this new prefix is just a square (but the same applies to python prefix..).

**Comment by vadviktor** [comment]
Hey @dawikur I'm fine to hold off with this.
The unicode I chose for Elixir is https://unicode-table.com/en/1F4A7/ (droplet).
I mostly use Fira Code font in the terminal but even Droid and Dejavu prints it fine. Using Konsole as terminal btw.

Any more I can help?

**Comment by dawikur** [comment]
OK, #188 landed successfully. Can you rebase with proper changes?

**Comment by vadviktor** [comment]
@dawikur conflict resolved, ready to go :)

File: README.md [hunk 1]
```diff
@@ -170,6 +171,15 @@ NOTE: You do not need to specify *end* segment - it will be added automatically.
 |`BULLETTRAIN_RUBY_FG`|`white`|Foreground color
 |`BULLETTRAIN_RUBY_PREFIX`|`"♦"`|Prefix of the segment
 
+### Elixir
+
+|Variable|Default|Meaning
+|--------|-------|-------|
+|`BULLETTRAIN_ELIXIR_SHOW`|`true`|Show/hide that segment
```
Comment (dawikur) [inline README.md]: Please remove  the `_SHOW` variable accordingly to #188.

Comment (vadviktor) [inline README.md]: @dawikur yes, good catch. I'll come back to this shortly.

Comment (dawikur) [inline README.md]: In README there is still one, sorry :)

**Comment by vadviktor** [comment]
You are a true friend @dawikur :) I'll have a more  thorough look bit later. Thanks for being sharp.

**Comment by dawikur** [comment]
I think it looks good.
@caiogondim what do you think?

**Comment by caiogondim** [comment]
Looks good to me =)

**Pull request merged**